assets/i18n/en.json

```diff
@@ -1,5 +1,6 @@
 {
   "I18N_ABOUT_PAGE_TITLE": "About us - Oppia",
+  "I18N_ACTION_APPLY_TO_TEACH_WITH_OPPIA": "Apply to Teach With Oppia",
```

Comment:
Thanks! Noting responses from @mit0110 here too, for future reference:
- Aplica para enseñar con Oppia
- Aplica para "Teach with Oppia"